src/model/modelinfo/ModelInfo.ts

```diff
@@ -0,0 +1,42 @@
+import TypeInfo, { ITypeInfoXml } from "./TypeInfo";
+
+export interface IModelInfoXml {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Interface name must not be prefixed with "I".](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5691315)